tests/cypress/integration/skeletons_pipeline.js

```diff
@@ -166,14 +169,39 @@ context('Manipulations with skeletons', () => {
             cy.get(selector).should('not.exist');
         }
 
-        it('Creating and removing a skeleton shape', () => {
+        it.skip('Creating and removing a skeleton shape', () => {
             createSkeletonObject('shape');
             deleteSkeleton('#cvat_canvas_shape_1', 'shape', false);
             cy.removeAnnotations();
         });
 
-        it('Creating and removing a skeleton track', () => {
+        it('Creating, re-drawing, and removing a skeleton track', () => {
             createSkeletonObject('track');
+
+            // redraw a tracked shape on the latest frame
+            const REDRAW_MARGIN = 400;
+            let prevX = Number.MAX_SAFE_INTEGER;
+            let prevY = Number.MAX_SAFE_INTEGER;
+            cy.goCheckFrameNumber(imageParams.count - 1);
+            cy.get('#cvat_canvas_shape_1').within(() => {
+                cy.get('rect').then(($rect) => {
+                    prevX = +$rect[0].getAttribute('x');
+                    prevY = +$rect[0].getAttribute('y');
+                });
+            });
+            cy.get('#cvat_canvas_shape_1').trigger('mousemove').should('have.class', 'cvat_canvas_shape_activated');
+            cy.get('body').trigger('keydown', { keyCode: 78, code: 'KeyN', shiftKey: true });
+            cy.get('.cvat-canvas-container')
+                .click(skeletonPosition.xtl + REDRAW_MARGIN, skeletonPosition.ytl + REDRAW_MARGIN)
+                .click(skeletonPosition.xbr + REDRAW_MARGIN, skeletonPosition.ybr + REDRAW_MARGIN);
+            cy.get('.cvat-cursor-control').should('have.class', 'cvat-active-canvas-control');
+            cy.get('#cvat_canvas_shape_1').within(() => {
+                cy.get('rect').then(($rect) => {
+                    expect(+$rect[0].getAttribute('x')).to.be.gt(prevX);
+                    expect(+$rect[0].getAttribute('y')).to.be.gt(prevY);
```

Comment:
Maybe check for more specific number than just greater? Something like `to be in [prevX+MARGIN/2; prevX+MARGIN*1.5 ]`

Reply:
It is not possible here, there are some translated coordinates (with offset, scale, etc)